packages/mobile/src/geth/saga.ts

```diff
@@ -65,7 +78,7 @@ function* waitForGethInstance() {
   try {
     const fornoMode = yield select(fornoSelector)
     // get geth without syncing if fornoMode
-    const gethInstance = yield call(getGeth, !fornoMode)
+    const gethInstance = yield call(initGeth, !fornoMode)
     if (gethInstance == null) {
       throw new Error('geth instance is null')
```

Comment:
Should this error message change given you're now returning `null` if there's already a geth creation in progress?

Reply:
Good question. I believe the checking for specifically null instead of all falsy values was just an oversight. I'll fix this.